.controlplane/Dockerfile_from_base

```diff
@@ -0,0 +1,17 @@
+FROM ahangarha/rwrt-base:latest
```

Comment:
WTFs:
1. personal name.
2. need comments on why file exists and how to use

Reply:
The reason behind this was that before pushing this base image to the official docker hub of the company, I could demonstrate deployment. I needed a review on the implementation of this Docker base image. Now I got it and now I can move forward.